modules/ROOT/pages/dashboard-custom-config.adoc

```diff
@@ -159,23 +141,28 @@ image::dashboard-custom-config-dup-delete.png[Configure a Chart]
 * xref:dashboard-custom-config-text.adoc[Configuring Text Panels]
 
 [[dup_dashboard]]
-== Duplicating a Custom Dashboard
+== Duplicate a Custom Dashboard
 
 To create a copy of a custom dashboard:
 
 . From Custom Dashboards, open a custom dashboard.
-. Click three vertical dots on the top-right of the dashboard, and select Duplicate.
+. Click the three vertical dots on the top-right of the dashboard, and select *Duplicate*.
 +
 [[dup_delete_dashboard]]
 image::dashboard-custom-dup-delete.png[Duplicate Dashboard]
 +
-. Save a New Name for the dashboard.
+. Save a *New Name* for the dashboard.
 +
-The UI will open to your copy of the dashboard.
+The UI opens to your copy of the dashboard.
 
 [[delete_dashboard]]
-== Deleting a Custom Dashboard
+== Delete a Custom Dashboard
 
 . From Custom Dashboards, open a custom dashboard.
 . Click three vertical dots on the top-right of the dashboard, and select <<dup_delete_dashboard, Delete>>.
-. Confirm your deletion.
\ No newline at end of file
+. Confirm your deletion.
```

Comment:
I love this! concise, friendly, and if they change the control, it's future-proofed!